stac_pydantic/extensions.py

```diff
@@ -9,10 +10,21 @@
 from stac_pydantic.item import Item
 
 
+@lru_cache(maxsize=128)
+def _fetch_schema(url: str) -> dict:
```

Comment:
NIT - please rename to `_fetch_and_cache_schema` so its obvious from the call that caching is happening

Reply:
Yes, done.